README.md

```diff
@@ -7,18 +7,20 @@ You can also use it as a sidebar that you can put on right or left side
 ![SublimeFileBrowser Screenshot2](http://cl.ly/image/0Z2U062k3l3p/Screen%20Shot%202014-01-24%20at%2011.26.53.png)
 
 ## Installation
-You can install via [Sublime Package Control](http://wbond.net/sublime_packages/package_control)  
```

Comment:
You’ve removed several double spaces, please put them back (they are for line breaks)